tox.ini

```diff
@@ -16,6 +16,8 @@ deps =
     Pillow != 3.0.0
     leveldb
     -rtests/requirements-py2.txt
+    reppy
+    robotexclusionrulesparser
```

Comment:
Instead of adding them here and below, add them to the corresponding test requirement files (`tests/requirements-py2.txt`, `tests/requirements-py3.txt`), as they are necessary to run the tests.